### ISSUE TITLE
Show all chatlines from client

### ISSUE THREAD

**Issue by wassie** [AoKMiKeY/Echelon#2]
It would be great if it would be possible to see all chats from a specific client

**Comment by AoKMiKeY** [comment]
I will be improving this along side your last suggestion https://github.com/AoKMiKeY/Echelon/issues/1 thanks :)

**Comment by wassie** [comment]
I have noticed that when I click on a username, you are able to see all chatlog from this user. So I think this is already possible. If you agree, we could close this issue.

**Comment by MichaelHillcox** [comment]
@wassie Yeah, you can do that but I do not think it shows you them all. For instance on my servers we have a huge amount of chatlogs. Our database has over 2mill entries but it does not reflect on the players. I will keep this open because I want to make it so you can click on the players name in the global  chatlog and it will take you to that point in there history of chatlogs. The chatlog will also be able to be searched though. Lots of general improvements to the chat logs :).

**Comment by wassie** [comment]
@MichaelHillcox great Michael, I'm really curious when you guys publish a new release!
